src/main/java/com/mjd/bank/dtos/request/CreateRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.mjd.bank.dtos.request;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.mjd.bank.entities.AccountType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain=true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CreateRequest {
+    private String type;
+}
```

Comment:
En el request body deberíamos incluir el owner Id. Esto se hace para poder comparar el owner Id que viene en el body con el ownerId que viene en el header. Una persona no debería poder crear una cuenta para otro usuario. Si no hacemos esta comprobación que te digo cualquiera podría crear una cuenta para cualquiera. Además debería llamarse diferente la clase. Algo como CreationRequest en vez de CreateRequest

```suggestion
public class CreationRequest {
    private String type;
    private Long ownerId;
}
```

Reply:
Entendido